TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -347,6 +349,9 @@ bool TTSSpeaker::waitForStatus(GstState expected_state, uint32_t timeout_ms) {
 // GStreamer Releated members
 void TTSSpeaker::createPipeline() {
     m_isEOS = false;
```

Comment:
as of now, it looks like if the end points are changed at runtime through "setttsconfiguration" method, the pipeline is not updated i.e it will keep on reaching out to older endpoint until a pipeline error occurs. Can we trigger pipeline update when the end point changes?